src/universal/components/PushButton/PushButton.js

```diff
@@ -1,42 +1,49 @@
-import React, { Component, PropTypes } from 'react';
-import look, { StyleSheet } from 'react-look';
+import React, {PropTypes} from 'react';
+import look, {StyleSheet} from 'react-look';
 import theme from 'universal/styles/theme';
 
 const combineStyles = StyleSheet.combineStyles;
 
 let styles = {};
 
-@look
-// eslint-disable-next-line react/prefer-stateless-function
-export default class PushButton extends Component {
-  static propTypes = {
-    disabled: PropTypes.bool,
-    keystroke: PropTypes.string,
-    label: PropTypes.string,
-    onClick: PropTypes.func,
-    size: PropTypes.oneOf([
-      'default',
-      'large'
-    ])
-  }
+const PushButton = (props) => {
+  const { disabled, keystroke, label, onClick, size } = props;
+  const largeStyles = combineStyles(styles.button, styles.buttonLarge);
+  const buttonStyles = size === 'large' ? largeStyles : styles.button;
+  const handleClick = (e) => {
```

Comment:
Yeah I think we are using a `<button/>` element so shouldn’t have too many weird browser UI side effects.